workbench-session-init/run.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -e
```

Comment:
If we can, I'd love for this to more strongly error out when there is a failure in the script.
```suggestion
set -euo pipefail
```